src/bp/ui-studio/src/web/views/OneFlow/diagram/index.tsx

```diff
@@ -634,6 +636,15 @@ class Diagram extends Component<Props> {
     return targetModel instanceof StandardNodeModel || targetModel instanceof SkillCallNodeModel
   }
 
+  onDiagramDoubleClick = (event: MouseEvent) => {
```

Comment:
aligned with this, it would be much useful to go back to calling flow when focused on reusable.

Reply:
> While debugging, debugging there's a double `jump into`
> ![Screen Shot 2020-10-06 at 8 38 12 AM](https://user-images.githubusercontent.com/955524/95202614-8e1f0180-07af-11eb-94e3-28f5a709daed.png)

Yup, it's when you enter a subworkflow then exit it during the same "turn", you "jump into" when going from the subworkflow node to the reusable, then you techhnically "jump out" from the success node to the subworkflow node. Should filter out one of them